### ISSUE TITLE
Pharmacies page does not open as top leve page but as Dialog

### ISSUE THREAD

**Issue by cygairko** [gematik/E-Rezept-App-Android#14]
When opening the Pharmacies page through the bottom navigation it will open up as a Dialog. This kind of behavior is not intended with a bottom navigation. Instead it should open a top level page such as correctly done on orders page for example.

Side note: Also the Settings page opens with a different animation. Looks like only Prescription and Orders pages are working correctly.
```[tasklist]
### Tasks
- [ ] Do not open pharmacies as dialog when opening from bottom navigation
```

**Comment by dineshvg** [comment]
Sorry about the late reply, we have started refactoring some of the existing code, and we will do this also as a part of that. Thanks for the issue

**Comment by dineshvg** [comment]
This will come with the next release.

**Comment by dineshvg** [comment]
This has been fixed in the 1.22.1 playstore release. We will update the code here soon with a new release.
